foreman/data_refinery_foreman/surveyor/test_sra.py

```diff
@@ -112,6 +112,34 @@ def test_srp_survey(self, mock_send_task):
         self.assertEqual(experiment.alternate_accession_code, None)
         self.assertEqual(len(samples), 9)
 
+    @vcr.use_cassette("/home/user/data_store/cassettes/surveyor.sra.survey_file_report.yaml")
+    @patch("data_refinery_foreman.surveyor.external_source.send_job")
+    def test_survey_file_report(self, mock_send_job):
```

Comment:
Do we have a test that makes sure that unmated reads are detected properly?

Reply:
Yes, that's this one. I've renamed it to make that more clear.